plugin/src/main/java/net/dzikoysk/funnyguilds/FunnyGuilds.java

```diff
@@ -2,14 +2,12 @@
 
 import com.google.common.collect.ImmutableSet;
 import eu.okaeri.configs.exception.OkaeriException;
-import java.io.File;
```

Comment:
Dear @Szczurowsky,

We wanted to take a moment to express our gratitude for your contribution in deleting another feature from our project, Funnyguilds. Your help in simplifying our codebase is much appreciated, and we're thankful for your hard work and dedication to the project.

However, we have noticed that you've been reorganizing imports, and we wanted to bring to your attention that it's causing a bit of a problem. Our ability to track who added what to files is getting messed up, which could lead to issues down the line.

We wanted to kindly ask that you refrain from reorganizing imports in the future, to help us maintain our codebase more efficiently. We value your contributions to our project, and we want to make sure that we're all working in sync to achieve our goals.

We recognize that your efforts were crucial in making this happen, and we're grateful for your contribution. Your willingness to lend a hand and help us achieve our goals is truly appreciated, and we're lucky to have you as a member of our community.

Once again, thank you for your hard work and dedication to the project. We're excited to see what the future holds.

Best regards,
Bopke

Reply:
Hi Bopke,

Thanks for mentioning the problem. I deeply understand problematic of mess in imports. However I believe that deleting unused one is also a part of keeping 'em clean and readable.

Best regards,
Szczurowsky